org.eclipse.m2e.jdt.tests/src/org/eclipse/m2e/jdt/tests/JavaConfigurationTest.java

```diff
@@ -124,6 +124,27 @@ public void testSkipNone() throws CoreException, IOException, InterruptedExcepti
 		assertEquals(1, classpathEntriesCount(project, TEST_RESOURCES));
 	}
 
+	
+	@Test
+	public void testComplianceVsEnablePreviewSettings() throws CoreException, IOException, InterruptedException {
+		IJavaProject project = importResourceProject("/projects/compilerEnablePreviewSettings/pom.xml");
+		assertEquals("10", project.getOption(JavaCore.COMPILER_SOURCE, false));
+		assertEquals("10", project.getOption(JavaCore.COMPILER_CODEGEN_TARGET_PLATFORM, false));
+		assertEquals("10", project.getOption(JavaCore.COMPILER_COMPLIANCE, false));
+		assertEquals(JavaCore.DISABLED, project.getOption(JavaCore.COMPILER_PB_ENABLE_PREVIEW_FEATURES, false));
```

Comment:
Shouldn't this be something like enabled?
```suggestion
		assertEquals(JavaCore.ENABLED, project.getOption(JavaCore.COMPILER_PB_ENABLE_PREVIEW_FEATURES, false));
```